verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -62,27 +62,31 @@ export default class CasesController {
         }
     };
 
+    upsert = async (req: Request, res: Response): Promise<void> => {
+        if (!(await this.geocode(req))) {
+            res.status(404).send(
+                `no geolocation found for ${req.body['location']?.query}`,
+            );
+            return;
+        }
+        try {
+            const response = await axios.put(
+                this.dataServerURL + '/api' + req.url,
+                req.body,
+            );
+            res.json(response.data);
```

Comment:
Not necessary now but it seems that returning 200 when updated and 201 when created is standard procedure. To keep in mind I guess.

Reply:
Yep -- wasn't rocking the boat here. We should fix that at some point, though.